src/main/java/kostka/moviecatalog/factory/LoginFactory.java

```diff
@@ -0,0 +1,21 @@
+package kostka.moviecatalog.factory;
+
+import java.util.Collection;
+
+public final class LoginFactory {
+
+    public LoginFactory() {
```

Comment:
If this is a utility class, private could be preferred by convention.

Reply:
right :)